{
  "modified": "2024-06-25T13:38:25Z",
  "published": "2024-06-25T13:38:25Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-5510",
  "aliases": [
    "SNYK-PYTHON-PIPCRYPTLIBERYV2-5932114"
  ],
  "summary": "Malicious code in pipcryptliberyv2 (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pipcryptliberyv2",
        "purl": "pkg:pypi/pipcryptliberyv2"
      },
      "versions": [
        "1.1.0",
        "1.0.0"
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-PIPCRYPTLIBERYV2-5932114"
    },
    {
      "type": "ARTICLE",
      "url": "https://medium.com/checkmarx-security/the-evolutionary-tale-of-a-persistent-python-threat-d19196d4ed15"
    },
    {
      "type": "ADVISORY",
      "url": "https://research.jfrog.com/malicious-packages/"
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-04292",
        "import_time": "2024-06-28T02:49:42.90579432Z",
        "modified_time": "2024-06-25T13:38:25Z",
        "sha256": "bedb95b78a79bd59f32c8d55433389a79c595a831433da4c9b198a7f466fb4ef",
        "source": "reversing-labs",
        "versions": [
          "1.1.0",
          "1.0.0"
        ]
      }
    ]
  }
}
